database/database.go

```diff
@@ -0,0 +1,255 @@
+package database
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/NebulousLabs/skynet-accounts/build"
+	"github.com/NebulousLabs/skynet-accounts/user"
+
+	"gitlab.com/NebulousLabs/errors"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+var (
+	// SkynetDBHostEV holds the name of the environment variable for DB host.
+	SkynetDBHostEV = "SKYNET_DB_HOST"
+	// SkynetDBPortEV holds the name of the environment variable for DB port.
+	SkynetDBPortEV = "SKYNET_DB_PORT"
+	// SkynetDBUserEV holds the name of the environment variable for DB username.
+	SkynetDBUserEV = "SKYNET_DB_USER"
+	// SkynetDBPassEV holds the name of the environment variable for DB password.
+	SkynetDBPassEV = "SKYNET_DB_PASS"
```

Comment:
I'd probably call them `EnvDBHost`, `EnvDBPort` etc. to make it easier to find using autocomplete. Also `Skynet` is redundant within this application. In the string it's fine.

Reply:
Makes sense.